core/src/main/scala/flatgraph/DiffGraphApplier.scala

```diff
@@ -12,7 +12,7 @@ object DiffGraphApplier {
     graph: Graph,
     diff: DiffGraphBuilder,
     schemaViolationReporter: SchemaViolationReporter = new SchemaViolationReporter
-  ): Unit = {
+  ): Int = {
```

Comment:
Our future selves are demanding to add some comments like `/** returns the number of modifications that were successfully applied to the graph */` to methods like this, which only return an Int that could mean anything or nothing.